charts/intel-gaudi-resource-driver/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
Will the resource driver work without service account? Why are we giving the choice to not install it ?